pkg/cmd/cli/cmd/helper.go

```diff
@@ -80,3 +80,22 @@ func selectString(s, spec string) bool {
 	}
 	return match
 }
+
+// parseValue parses the value and returns namespace, resource and the
+// value (resource name) itself. The valid syntax is:
+// oc process mytemplate - implicit namespace (current), implicit resource (template)
+// oc process template/mytemplate - implicit namespace (current), explicit resource
+// oc process ns/template/mytemplate - explicit namespace, explicit resource
+// oc process ns//mytemplate - explicit namespace, implicit resource (template)
+func parseValue(v string) (ns, resource, name string) {
+	parts := strings.Split(strings.TrimSpace(v), "/")
+	switch len(parts) {
+	case 3:
+		return parts[0], parts[1], parts[2]
+	case 2:
+		return "", parts[0], parts[1]
+	case 1:
+		return "", "", parts[0]
```

Comment:
need a default case that returns "","","",false so we don't accidentally accept bad data